README.md

```diff
@@ -64,18 +88,12 @@ Install the following prerequisites:
 - [Docker](https://www.docker.com/products/docker-desktop/)
   - **Important**: Ensure Docker is running before running any `azd` provisioning / deployment commands.
 
-##### Project Initialization
+Then, run the following commands to get the project on your local environment:
 
-1. Create a new folder and switch to it in the terminal
-1. Run `azd auth login`
-1. Run `azd init -t azure-search-openai-demo-csharp`
-
-> **Note**<br>
-> Your Azure Account must have `Microsoft.Authorization/roleAssignments/write` permissions, such as [User Access Administrator](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#user-access-administrator) or [Owner](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#owner).
+   1. Run `azd auth login`
+   1. Run `azd env new azure-search-openai-demo-csharp`
```

Comment:
Don't you need to do `azd init` to bring the code down? Or is the expectation that they are going to `git clone` this repo rather than use it as an azd template?

Reply:
Good question. It depends. If you haven't cloned the repo, calling `azd init` does that for you. Otherwise you can take the approcah you suggested.